src/modbus-tcp-client.js

```diff
@@ -146,14 +154,14 @@ module.exports = stampit()
 
             reqId += 1;
 
-            var head = Buffer.allocUnsafe(7)
+            var head = Buffer.allocUnsafe(7);
 
-            head.writeUInt16BE(reqId, 0)
-            head.writeUInt16BE(this.protocolVersion, 2)
-            head.writeUInt16BE(pdu.length + 1, 4)
-            head.writeUInt8(this.unitId, 6)
+            head.writeUInt16BE(reqId, 0);
```

Comment:
I am using syntastic for vim that works great with `standard`. I like the minimalist code this encourages and I see more and more projects are migrating, however at the end is a matter of taste.

However I would suggest migrating the project to use strict mode in general 

Another topic would be ES6 features (e.g. `let` and `const` scoping and others) that improve readability and explicitness a lot, however drop compatibility to everything older than Node5 why I understand when open source projects hesitate to use that..